admin/homebrew.py

```diff
@@ -174,5 +183,57 @@ def install
            resources=get_resource_stanzas(dependency_graph),
            dependencies=get_formatted_dependency_list(dependency_graph))
 
+
+def verify_recipe(recipe, version):
+    """
+    Revert the Yosemite VM to a state where homebrew has just been
+    installed, start the machine and ssh into it. Then start a VM and test a
+    Homebrew script in that VM.
+
+    The machine must have:
+        * VMWare Fusion installed,
+        * A VMWare OS X VM available at a particular location
+
+    The VM must have:
+        * A snapshot with a particular name,
+        * Homebrew installed and available
+
+    :param str recipe: The contents of a Homebrew recipe to be installed.
+    :param str version: The version of Flocker which the Homebrew recipe will
+        install.
+    """
+    YOSEMITE_VMX_PATH = "{HOME}/Desktop/Virtual Machines.localized/OS X 10.10.vmwarevm/OS X 10.10.vmx".format(HOME=environ['HOME'])
+    # This can likely be found another way, but I got it from within the VM:
+    # System Preferences > Network
+    VM_ADDRESS = "172.18.140.54"
+
+    # XXX Requires https://github.com/msteinhoff/vmfusion-python/pull/4 to be
+    # merged
+    vmrun.revertToSnapshot(YOSEMITE_VMX_PATH, 'homebrew-clean')
+    vmrun.start(YOSEMITE_VMX_PATH, gui=False)
+
+    recipe_path = "flocker-{version}.rb".format(version=version)
```

Comment:
@jongiddy and @dwgebler If you want to use this, it might be useful to know that you could replace this with a URL to the recipe, if you get rid of the `Put` below.